app/javascript/article-form/components/__tests__/ArticleCoverImage.test.jsx

```diff
@@ -195,20 +207,20 @@ describe('<ArticleCoverImage />', () => {
     });
 
     it('does not contain the file input for uploading an image used only in the web browser experience', async () => {
-      const { queryByText } = render(
+      const { queryByTestId } = render(
         <ArticleCoverImage mainImage="" onMainImageUrlChange={jest.fn()} />,
       );
-      expect(queryByText(/Upload an image/i)).not.toBeInTheDocument();
+      expect(queryByTestId('cover-image-input')).not.toBeInTheDocument();
```

Comment:
Just wondering why we're not querying this one by label text or role instead?

Reply:
I tested if it was possible to find the element by role, but couldn't find it so I used `queryByTestId` instead. 

But I just tested it, and I could've used the `queryByLabelText`! Good call out. I'm in the midst of submitting another PR right now, so I can fix this in there :)